lib/assets/SnapshotAlertManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SnapshotAlertManager.swift
+//
+//  Created by Maxime Britto on 06/01/2016.
+//  Copyright © 2016 dev2a. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+func automaticallyDismissAlerts(testCase:XCTestCase) -> NSObjectProtocol {
+    return SnapshotAlertManager.automaticallyDismissAlerts(testCase)
+}
+
+func stopDismissingAlerts(testCase:XCTestCase) {
+    SnapshotAlertManager.stopDismissingAlerts(testCase)
+}
+
+func waitForAlertsToBeDismissed() {
+    SnapshotAlertManager.waitForAlertsToBeDismissed()
+}
+
+class SnapshotAlertManager: NSObject {
+    static var alertInterruptionMonitor:NSObjectProtocol?
+
+    class func stopDismissingAlerts(testCase:XCTestCase) {
+        if let previousDismissHandler = alertInterruptionMonitor {
+            testCase.removeUIInterruptionMonitor(previousDismissHandler)
+        }
+    }
+
+    class func automaticallyDismissAlerts(testCase:XCTestCase) -> NSObjectProtocol
+    {
+        SnapshotAlertManager.stopDismissingAlerts(testCase)
+        alertInterruptionMonitor = testCase.addUIInterruptionMonitorWithDescription("Snapshot alert auto hide") { (alert: XCUIElement) -> Bool in
```

Comment:
well the lack of docs is awful, but the functionality is sweet. is that how `XCUIApplication().tap()` works below as well?